lib/features/pokemon_details/pokemon_details_connector.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:pokedex_async_redux/features/pokemon_details/pokemon_details.dart';
+import 'package:pokedex_async_redux/state/state.dart';
+import 'package:async_redux/async_redux.dart';
+import 'package:flutter/material.dart';
+
+class PokemonDetailsConnector extends StatelessWidget {
+  const PokemonDetailsConnector({
+    required this.pokemonName,
+    required this.pokemonImage,
+    super.key,
+  });
+
+  final String pokemonName;
```

Comment:
why not pass the whole pokemon object instead since these properties u indicated are all from pokemon model? 🙂